_gsocproposals/2022/proposal_ESCAPE-repository-as-a-service.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Repository as a Service
+layout: gsoc_proposal
+project: ESCAPE-OSSR
+year: 2022
+difficulty: medium
+duration: 300
+mentor_avail: June-September
+organization: ESCAPE
+---
+
+## Description
+
+The [ESCAPE project](https://projectescape.eu/) is building an Open Source and collaborative 
+environment in where to connect astronomy, astroparticle and particle physics project. This 
+research environment is composed of the ESCAPE services. In this context, the Open-source 
+Science and Service Repository (OSSR) provides a trusted repository that will be accessible 
+through the ESFRI Science Analysis Platform (ESAP).
+
+The successful candidate will be integrated into the ESCAPE project to develop a JupyterLab 
+extension to create an user friendly connection between the repository and the analysis platform 
+(the computing resources). This plugin will simplify the access to Zenodo (the Open Source 
+Repository) that provides an open archive for scientists to share their analysis software 
+and data.
+ 
+
+## Task ideas and expected Results
+
+* Create a JupyterLab extension that connects the Jupyter environment and Zenodo (Repository) using the  
+eOSSR python library to display the repository records.
+* Participate in the development of the eOSSR library to integrate the developed extension. 
+* Bring back the record content to the Jupyter environment using the developed extension.
+* Investigate the possibility of installing the different software records available in the repository 
+and find solutions to the different software languages libraries.
+* Define the needed stages to publish back into Zenodo and the ESCAPE community.
+
+
```

Comment:
It's recommended to add at least one sentence about what the student will learn.